src/corelibs/U2View/src/ov_assembly/AssemblyBrowser.h

```diff
@@ -54,11 +54,12 @@ class AssemblyBrowser : public GObjectView {
     bool checkValid(U2OpStatus &os);
 
     // from GObjectView
-    virtual void buildStaticToolbar(QToolBar *tb);
-    virtual void buildStaticMenu(QMenu *m);
-    virtual QVariantMap saveState();
-    virtual Task *updateViewTask(const QString &stateName, const QVariantMap &stateData);
-    virtual OptionsPanel *getOptionsPanel();
+    void buildStaticToolbar(QToolBar *tb) override;
+    void buildMenu(QMenu *menu, const QString &type) override;
+
+    QVariantMap saveState() override;
+    Task *updateViewTask(const QString &stateName, const QVariantMap &stateData) override;
+    OptionsPanel *getOptionsPanel() override;
```

Comment:
`OptionsPanel *getOptionsPanel() const override;`

Reply:
I can't make it const because it is not designed to be const. I think name 'get' is misleading here but this kind of refactoring is out of scope of this huge PR: I just added 'override'.